opensrp-chw/src/main/java/org/smartregister/chw/actionhelper/ChildHVProblemSolvingHelper.java

```diff
@@ -47,11 +47,7 @@ public String postProcess(String jsonPayload) {
 
     @Override
     public BaseAncHomeVisitAction.Status evaluateStatusOnPayload() {
-        if (StringUtils.isBlank(child_playing_challenge)) {
-            return BaseAncHomeVisitAction.Status.PENDING;
-        }
```

Comment:
@devcolonel This handled "Null point exception" onBackpress without filling the module form